src/app/components/source/MeComponent.js

```diff
@@ -0,0 +1,131 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { browserHistory } from 'react-router';
+import { FormattedMessage } from 'react-intl';
+import Tabs from '@material-ui/core/Tabs';
+import Tab from '@material-ui/core/Tab';
+import cx from 'classnames/bind';
+import UserEmail from '../user/UserEmail';
+import UserInfo from './UserInfo';
+import UserPrivacy from './UserPrivacy';
+import UserSecurity from './UserSecurity';
+import UserInfoEdit from './UserInfoEdit';
+import { can } from '../Can';
+import PageTitle from '../PageTitle';
+import CheckContext from '../../CheckContext';
+import SwitchTeamsComponent from '../team/SwitchTeamsComponent';
+import styles from './User.module.css';
+
+class MeComponent extends React.Component {
```

Comment:
I see this is a class component - I'm guessing it's because we are moving functionality from `UserComponent` and it's more efficient to move it without also refactoring at the same time. Just wanted to flag that this is a (sort of) new class component we are introducing.

Reply:
exactly @dariusk I cloned `UserComponent` to be `MeComponent` and removed unneeded code from `UserComponent`

Reply:
Okay, sounds good!